Ball of Duty Server/Domain/Entities/Character.cs

```diff
@@ -45,18 +47,34 @@ public void AddKill(Character victim)
             {
                 HighScore = Score;
             }
+            UpdateServerStats();
             NotifyObservers(Observation.ACQUISITION_OF_GOLD, victim);
         }
 
+        public void DecayAndRegen()
+        {
+            DecayScore();
+            RegenHealth();
+        }
+
+        public void RegenHealth()
+        {
+            if (Health.Value < Health.Max)
```

Comment:
what if you have 1 health less than max, then you will suddenly have more health than max when this method executes..